src/components/AccordionTableRow/Example.md

```diff
@@ -19,7 +19,7 @@ const rowRenderer = ({ cells, handleClick, active }) => (
 function getData(num) {
   return Array.from(new Array(num), (el, ind) => ({
     key: `Key #${ind}`,
-    val: Math.random()
+    val: (ind * 356)
```

Comment:
If its really about stabilizing ("aka" seeding) random numbers look, e.g., at

```js
var seed = 1;
function random() {
    var x = Math.sin(seed++) * 10000;
    return x - Math.floor(x);
}
```

There are other ways, but this is one of the simplest which should suffice here.